packages/onboarding/src/navigator/navigator-buttons/index.tsx

```diff
@@ -54,7 +54,8 @@ export const NavigationButtonAsItem = ( { className, ...props }: Props ) => {
 	return (
 		<NavigatorButton
 			as={ GenericButton }
-			className={ classnames( 'navigator-button__wrapper', 'navigator-button', className ) }
+			className={ classnames( 'navigator-button', className ) }
+			wrapperClassName="navigator-button__wrapper"
```

Comment:
@arthur791004 `wrapperClassName` is no longer an accepted prop -- it causes a type error in the other PR. I commented out temporarily. Could you find another solution?

Reply:
it does seem to work without it:
<img width="1676" alt="Screenshot 2023-07-12 at 4 13 43 PM" src="https://github.com/Automattic/wp-calypso/assets/6265975/02dc0f09-9e69-4ca6-9ecb-759cbf4ea5fc">

Reply:
The issue is that the navigator button won't take the full width without that class name (compare to the “Save and Continue” button). Let me think about how to resolve it.